lib/ActivityStreams.js

```diff
@@ -443,6 +470,21 @@ ActivityStreams.prototype = {
   },
 
   /**
+    * Start a timer to fetch a new recommendation every hour. This will only
+    * run for those in the experiment
+    */
+  _refreshRecommendations(recommendationTTL) {
+    if (recommendationTTL) {
+      this._newRecommendationTimeoutID = setTimeout(() => {
```

Comment:
hmm i was looking at the RecommendationProvider.

You're also setting a timeout there:

https://github.com/mozilla/activity-stream/blob/790d04b0a4db4ba2c9dce16069f0e697c7dc6f9d/lib/RecommendationProvider.js#L23

This makes a network request